funcx_endpoint/funcx_endpoint/endpoint/endpoint_manager.py

```diff
@@ -144,6 +145,19 @@ def start_endpoint(self, name, endpoint_uuid, endpoint_config):
             print('3. Start the endpoint.')
             return
 
+        # Error handling for loading config
+        default_address = SourceFileLoader('config', endpoint_default_config.__file__).load_module().config.funcx_service_address
+        try:
+            if hasattr(endpoint_config.config, 'provider') or hasattr(endpoint_config.config, 'max_workers_per_node'):
+                self.logger.exception('Need to update the out of date config. Refer to https://funcx.readthedocs.io/en/latest/endpoints.html#configuring-funcx')
+                raise AttributeError
+            elif not endpoint_config.config.funcx_service_address == default_address:
+                self.logger.exception(f'Should use \'{default_address}\' in config as funcx_service_address')
+                raise AttributeError
```

Comment:
This may be unfriendly for people who want to set up a test deployment and test endpoint with the deployment?